.github/workflows/python-cross-platform-support.yml

```diff
@@ -0,0 +1,146 @@
+on:
+  workflow_call:
+
+defaults:
+  run:
+    working-directory: ./python
+
+jobs:
+  os_build_matrix:
+    name: Build ${{ matrix.os }} Secure Install Air Gap
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-latest, windows-latest, macos-latest] # Annoyingly, Github actions don't support anchors
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
```

Comment:
Ideally we'd have two matrices of operating systems and versions that we support, and we'd run the tests in the combinations of those environments, however, we do need to be cognizant to not overrun the free pipeline minutes we get. 

So, for now, I'm comfortable with this arrangement since we are exercising AaC in all OS environments (3.9 only) and all versions (linux only), which gives us a fairly comfortable coverage. If there are any issues that aren't covered by this spread, I'd expect it to be limited to dependencies or 3.10/3.11 features (which would break the 3.9 tests and compatibility).